containers/event-db-migrations/entry.sh

```diff
@@ -31,6 +32,7 @@
 # SKIP_GRAPHQL_INIT - If set, graphql will not be initialized (optional)
 # DEBUG - If set, the script will print debug information (optional)
 # DEBUG_SLEEP - If set, the script will sleep for the specified number of seconds (optional)
+# STAGE - The stage being run.  Currnelty only controls if stage specific data is applied to the DB (optional)
```

Comment:
I don't know what is the rationale for this but I imagine it is to not create an environment variable called environment?

Reply:
I see it's been used pretty much everywhere in the PR, so it might be a pain to change at this point. Based on the comment, though, it's not really clear that this basically means "environment" because that's what's getting passed to it (i.e. `dev`, `testnet`, etc.). So maybe just better documentation would be helpful.

For clarity, I'm thinking from the perspective of external users here.

Reply:
I agree with you. I've created an issue to track that (https://github.com/input-output-hk/catalyst-core/issues/498)